search-sentiments-map/test/trends-test.js

```diff
@@ -63,48 +51,39 @@ describe('Trends', function() {
     it('should retrieve the most recent datastore entry beyond RETRIEVE_RESULTS_TIME_MS',
         async function() {
       let results = await trends.retrieveGlobalTrendsForTimeRange(0);
-      assert.deepEqual(results, mockResult1half);
+      
```

Comment:
Delete this new line. Add a new line after line 58 instead.

Reply:
ok